src/iiptilesource.js

```diff
@@ -0,0 +1,319 @@
+/**
+ * OpenSeadragon - IIPTileSource
+ *
+ * Copyright (C) 2009 CodePlex Foundation
+ * Copyright (C) 2010-2024 OpenSeadragon contributors
```

Comment:
Might as well extend this to 2025.

Reply:
Yeah, I originally thought about doing that, but I saw all the other js files are still 2024. Anyway, now updated

Reply:
Yeah, we should get around to doing it for all of them, but we might as well at least do them as we see them.